lblod/ember-rdfa-editor-lblod-plugins
# Move rdfa-date plugin into variable plugin

### Overview
This PR removes the seperate rdfa-date plugin and merges it into the variable plugin.
It includes the following breaking changes:
  - Rename of variable date insert component from `VariablePlugin::Date::Insert` to `VariablePlugin::Date::InsertVariable` 
  - Rename of standalone date insert component from `RdfaDatePlugin::Insert` to `VariablePlugin::Date::Insert`
  - Removal of `RdfaDatePlugin::Card` component, replaced by `VariablePlugin::Date::Edit` component
  - Removal of `RdfaDatePlugin::Date` component, replace by `VariablePlugin::Date::Nodeview` component
  - Removal of `RdfaDatePlugin::DateTimePicker` component, replaced by `VariablePlugin::Date::DateTimePicker` component
  - Removal of `RdfaDatePlugin::HelpModal` component, replaced by `VariablePlugin::Date::HelpModal` component
  - The `date` node-spec and `dateView` node-view should now be imported from `@lblod/ember-rdfa-editor-lblod-plugins/plugins/variable-plugin/variables`

### How to test/reproduce
- Open the `besluit` or `regulatory statements` dummy route.
- Insert a date and edit it
- Check if the date persists correctly after reloading the document

### Challenges/uncertainties
- This PR removes the standalone `RdfaDatePlugin::Insert` component, as it became a bit redundant with the existence of the `VariablePlugin::Date::Insert` component which allows you to input a label and insert a date variable. 


### Checks PR readiness
